src/test/fuzz/util.h

```diff
@@ -338,9 +339,17 @@ inline void FillNode(FuzzedDataProvider& fuzzed_data_provider, CNode& node, cons
     }
 }
 
-inline void InitializeFuzzingContext(const std::string& chain_name = CBaseChainParams::REGTEST)
+template <class T = const BasicTestingSetup>
+std::unique_ptr<T> MakeFuzzingContext(const std::string& chain_name = CBaseChainParams::REGTEST, const std::vector<const char*>& extra_args = {})
 {
-    static const BasicTestingSetup basic_testing_setup{chain_name, {"-nodebuglogfile"}};
+    // Prepend default arguments for fuzzing
+    const std::vector<const char*> arguments = Cat(
+        {
+            "-nodebuglogfile",
+        },
+        extra_args);
+
+    return MakeUnique<T>(chain_name, arguments);
```

Comment:
nit: new code can use

```suggestion
    return std::make_unique<T>(chain_name, arguments);
```

Reply:
Fixed in #21003